cpp/build.sh

```diff
@@ -201,6 +201,39 @@ function install_libtorch() {
   cd "$BWD" || exit
 }
 
+function install_yaml_cpp() {
+  YAML_CPP_SRC_DIR=$BASE_DIR/third-party/yaml-cpp
+  YAML_CPP_BUILD_DIR=$YAML_CPP_SRC_DIR/build
```

Comment:
The build dir is not correct. It should be under dir _build/, not third-party dir. Please ref https://github.com/pytorch/serve/blob/cpp_backend/cpp/build.sh#L108